packages/proxy/lib/http/util/inject.ts

```diff
@@ -21,13 +23,17 @@ export function partial (domain, options: InjectionOpts) {
   // With useDefaultDocumentDomain=true we continue to inject an empty script tag in order to be consistent with our other forms of injection.
   // This is also diagnostic in nature is it will allow us to debug easily to make sure injection is still occurring.
   return oneLine`
-    <script type='text/javascript'>
+    <script type='text/javascript'${
+      cspNonce ? ` nonce="${cspNonce}"` : ''
```

Comment:
since we use this three places, can we refactor this to a function that returns the appropriate string when the `cspNonce` is passed in?